Sources/Helpers/OAUtilities.h

```diff
@@ -164,6 +164,6 @@ alpha:((float)((rgbValue & 0xFF000000) >> 24))/255.0]
 + (CGFloat) heightForHeaderViewText:(NSString *)text width:(CGFloat)width font:(UIFont *)font lineSpacing:(CGFloat)lineSpacing;
 
 + (NSDictionary *) getSortedVoiceProviders;
-+ (NSMutableAttributedString *) getStringWithBoldPart:(NSString *)wholeString mainString:(NSString *)ms boldString:(NSString *)bs lineSpacing:(CGFloat)lineSpacing;
++ (NSMutableAttributedString *) getStringWithBoldPart:(NSString *)wholeString mainString:(NSString *)ms boldString:(NSString *)bs lineSpacing:(CGFloat)lineSpacing highlightColor:(UIColor *)highlightColor;
```

Comment:
You know you could add a second method without the ```highlightColor``` parameter and simply call it with nil by default sou you do not need to change the method signature in the files where this added behavior is not required?